modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_form.rb

```diff
@@ -13,5 +13,283 @@ class PowerOfAttorneyForm < ApplicationRecord
     blind_index :city
     blind_index :state
     blind_index :zipcode
+
+    ##
+    # TODO: Can couple this to the schema involved in user input during POA 
```

Comment:
Is this TODO to move it to https://github.com/department-of-veterans-affairs/vets-json-schema ?